app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/cases_assigned_table.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.ReassignCaseController
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+
+@(title: String, cases: Seq[Case])(implicit request: AuthenticatedRequest[_], messages: Messages)
+
+@if(cases.nonEmpty) {
+    <h3 class="heading-small">@title</h3>
+    <table id="cases_list-table">
+        <thead>
+            <tr>
+                <th scope="col">@messages("cases.table.reference")</th>
+                <th scope="col">@messages("cases.table.status")</th>
+                <th scope="col">@messages("cases.table.elapsed-days")</th>
+                <th scope="col">@messages("cases.table.move")</th>
+            </tr>
+        </thead>
+        <tbody id="cases_list-body">
+        @for((c: Case, index: Int) <- cases.zipWithIndex) {
+            <tr id="cases_list-row-@index">
+                <th scope="row">
+                    <a id="cases_list-row-@{
+                        index
+                    }-reference" href="@uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.CaseController.trader(c.reference)">@c.reference</a>
+                </th>
+                <td id="cases_list-row-@{
+                    index
+                }-status">@c.status</td>
+                <td id="cases_list-row-@{
+                    index
```

Comment:
Not sure why these IDs are all multiline

Reply:
Prob due to IJ formatting.  Fixed